image_services/opera/README.md

```diff
@@ -0,0 +1,54 @@
+OPERA GIS Services
+=========
+**_A workflow for programatically standing up an OPERA Image Service_**
+-----
+
+## Steps for creating a service
+Below are the steps to create an image service using OPERA data. 
+
+### 1. Check permissions and server set up
+Prior to running the workflow, ensure that permissions are granted to pull from the EDC-hosted S3 bucket `asf-cumulus-prod-opera-products` and push and pull from a separate S3 bucket for overviews. 
```

Comment:
might be worth calling out that "pull permissions" means at least these:
```
ListBucket
GetBucketAcl
GetBucketLocation
GetObject
GetObjectVersion
GetObjectTagging
```

And "push permissions" is just:
```
PutObject
```

We could link to https://github.com/ASFHyP3/gis-services/blob/develop/image_server/cloudformation.yml#L145 as an example instead of listing everything here, but I'm on the fence about which is easier to understand.